src/node_messaging.cc

```diff
@@ -1065,7 +1065,11 @@ void MessagePort::MoveToContext(const FunctionCallbackInfo<Value>& args) {
         "The \"port\" argument must be a MessagePort instance");
   }
   MessagePort* port = Unwrap<MessagePort>(args[0].As<Object>());
-  CHECK_NOT_NULL(port);
+  if (port == nullptr) {
```

Comment:
```suggestion
  if (port == nullptr || port->IsHandleClosing()) {
```

I think this is basically what James is suggesting here, which seems fine to me (I don’t have a strong opinion here myself)

Reply:
Thank youuuuu!